frontend/src/features/map/Map.tsx

```diff
@@ -145,8 +145,15 @@ const Map: React.FC<MapProps> = ({ insets }) => {
 
 const styles = StyleSheet.create({
   locationButtonContainer: {
-    justifyContent: "flex-end",
-    alignItems: "flex-end",
+    flexDirection: 'row',
```

Comment:
I...really don't like this, to be honest. This really just does look like you hard-coded a position. There has to be something else that can be done here. Give me some time to find it.

Reply:
There isn't really any issue here in terms of a hardcoded position. We're using flex, align, justify content, and position normally.

I agree that "bottom" is sketchy, but without that, the button looks a bit out of place.